Features/huidig.feature

```diff
@@ -1,77 +1,124 @@
-dan worden alleen actuele objecten terugggeven waarvan de status NIET een van de volgende is:
+Wanneer parameter huidig=true wordt toegepast
+Dan worden alleen actuele objecten terugggeven waarvan de status NIET een van de volgende is:
 * naamgeving ingetrokken
 * woonplaats ingetrokken
 * plaats ingetrokken
 * niet gerealiseerd verblijfsobject
 * verblijfsobject ingetrokken
 * verblijfsobject ten onrechte opgevoerd
-* niet gerealiseerd pand,
+* niet gerealiseerd pand
 * pand gesloopt
 * pand ten onrechte opgevoerd
 
 #### Scenario:
-Gegeven een pand met verblijfsobjecten waaronder verblijfsobjecten met een eindstatus, (niet gerealiseerd verblijfsobject, of verblijfsobject ingetrokken of verblijfsobject ten onrechte opgevoerd)
-Wanneer de  API wordt bevraagd met parameter 'huidig'
-Dan bevat de response de verblijfsobjecten met een eindstatus niet en de verblijfsobjecten met de overige statussen (Verblijfsobject in gebruik, (niet ingemeten), Verblijfsobject in gebruik, Verblijfsobject buiten gebruik, tverbouwing verblijfsobject) wel
+Gegeven een pand met status 'pand in gebruik' met verblijfsobjecten waaronder verblijfsobjecten met een eindstatus, (niet gerealiseerd verblijfsobject, of verblijfsobject ingetrokken of verblijfsobject ten onrechte opgevoerd)
+Wanneer op het endpoint verblijfsobjecten de verblijfsobjecten in dat pand worden opgevraagd met de pandidentificatie zonder parameter 'huidig'
+Dan bevat de response alle verblijfsobjecten met een relatie met dat pand ongeacht de status van de verblijfsobjecten.
 
 #### Scenario:
-Gegeven een pand met een eindstatus
-Wanneer op het endpoint verblijfsobjecten de verblijfsobjecten in dat pand worden opgevraagd met de parameter 'huidig'
-Dan worden er geen verblijfsobjecten geretourneerd
+Gegeven een pand met status 'pand in gebruik' met verblijfsobjecten waaronder verblijfsobjecten met een eindstatus, (niet gerealiseerd verblijfsobject, of verblijfsobject ingetrokken of verblijfsobject ten onrechte opgevoerd)
+Wanneer op het endpoint verblijfsobjecten de verblijfsobjecten in dat pand worden opgevraagd met de pandidentificatie en de parameter 'huidig'
+Dan bevat de response de verblijfsobjecten met een eindstatus NIET en de verblijfsobjecten met de overige statussen (Verblijfsobject in gebruik, (niet ingemeten), Verblijfsobject in gebruik, Verblijfsobject buiten gebruik, tverbouwing verblijfsobject) wel
+
+Sporadisch komen statusconflicten tussen de verschillende objeten voor. In dat geval gelden de volgende scenario's:
+
+#### Scenario: Parameter huidig is ook van toepassing op de status van het object dat wordt gebruikt als queryparameter
+Gegeven een pand met een eindstatus met één of meer verblijfsobjecten met een huidige status
+Wanneer op het endpoint verblijfsobjecten de verblijfsobjecten in dat pand worden opgevraagd met de pandidentificatie en de parameter 'huidig'
+Dan wordt een lege collectie geleverd

```

Comment:
Als ik het goed heb beschrijf je hier hoe de api nu niet werkt. Of althans, waarvan we gezegd hebben dat de uitkomst nog niet specifiek vastligt. Ik denk dat hij op basis van de huidige code de verblijfsobjecten retourneert (en dat hij dus niet naar de eindstatus van het pand kijkt, alleen naar eventuele eindstatussen van verblijfsobjecten).

Als je dit gedrag zo wilt dan moeten we de werking van `huidig` langslopen bij alle endpoints.